c/src/point_one/polaris/polaris.c

```diff
@@ -855,14 +1082,12 @@ static int OpenSocket(PolarisContext_t* context, const char* endpoint_url,
   // Perform SSL handhshake.
   ret = SSL_connect(context->ssl);
   if (ret != 1) {
-    int err = SSL_get_error(context->ssl, ret);
-    P1_Print("SSL handshake failed to tcp://%s:%d, ssl_error=%d.\n",
-             endpoint_url, endpoint_port, err);
-#ifndef P1_FREERTOS
-    if (err == SSL_ERROR_SYSCALL) {
-      P1_PrintError("syscall error: ", errno);
-    }
-#endif
+#if !defined(P1_NO_PRINT)
+    char message[256];
```

Comment:
This is a lot of stack for freertos targets.

Reply:
True, though in practice I don't know if any FreeRTOS targets will also be using OpenSSL? We may be able to reuse the receive buffer in this context instead of declaring a local string. Let me see if that's possible.

Reply:
Yeah I think that's safe here.